app-modules/interaction/tests/Tenant/Filament/Actions/BulkCreateInteractionActionTest.php

```diff
@@ -0,0 +1,145 @@
+use AdvisingApp\StudentDataModel\Models\Student;
+
+use function Pest\Livewire\livewire;
+use function Tests\asSuperAdmin;
+
+it('shows the form and validation', function () {
+    asSuperAdmin();
+
+    $student = Student::factory()->create();
+
+    $request = collect(InteractionFactory::new()->create());
+
+    livewire(ListStudents::class)
+        ->mountTableBulkAction('createInteraction', [$student->getKey()])
+        ->setTableBulkActionData([
+            ...$request,
+        ])
+        ->callMountedTableBulkAction();
+})->with([
+    'initiative required' => [
+        InteractionFactory::new()->state(['interaction_initiative_id' => null]),
+        ['interaction_initiative_id' => 'required'],
+    ],
+    'driver required' => [
+        InteractionFactory::new()->state(['interaction_driver_id' => null]),
+        ['interaction_driver_id' => 'required'],
+    ],
+    'division required' => [
+        InteractionFactory::new()->state(['division_id' => null]),
+        ['interaction_initiative_id' => 'required'],
+    ],
+    'outcome required' => [
+        InteractionFactory::new()->state(['interaction_outcome_id' => null]),
+        ['interaction_outcome_id' => 'required'],
+    ],
+    'relation required' => [
+        InteractionFactory::new()->state(['interaction_relation_id' => null]),
+        ['interaction_relation_id' => 'required'],
+    ],
+    'status required' => [
+        InteractionFactory::new()->state(['interaction_status_id' => null]),
+        ['interaction_status_id' => 'required'],
+    ],
+    'type required' => [
+        InteractionFactory::new()->state(['interaction_type_id' => null]),
+        ['interaction_type_id' => 'required'],
+    ],
+    'start date required' => [
+        InteractionFactory::new()->state(['start_datetime' => null]),
+        ['start_datetime' => 'required'],
+    ],
+    'end time required' => [
+        InteractionFactory::new()->state(['end_datetime' => null]),
+        ['end_datetime' => 'required'],
+    ],
+    'subject required' => [
+        InteractionFactory::new()->state(['subject' => null]),
+        ['subject' => 'required'],
+    ],
+    'description required' => [
+        InteractionFactory::new()->state(['description' => null]),
+        ['description' => 'required'],
+    ],
+]);
+
+it('can successfully create bulk interaction with student', function () {
+    asSuperAdmin();
+
+    $student = Student::factory()->create();
+
+    $request = collect(InteractionFactory::new()->create([
+        'interactable_id' => $student->getKey(),
+        'interactable_type' => $student->getMorphClass(),
+    ]));
+
+    livewire(ListStudents::class)
+        ->mountTableBulkAction('createInteraction', [$student->getKey()])
+        ->setTableBulkActionData([
+            ...$request,
+        ])
+        ->callMountedTableBulkAction()
+        ->assertHasNoTableBulkActionErrors()
+        ->assertSuccessful()
+        ->assertNotified();
```

Comment:
You should expect/assert after this that the correct record exists in the database.